### DISCUSSION TITLE
v20.11.0

### DISCUSSION THREAD

**Discussion by colinmollenhour** [OpenMage/magento-lts#4426] [Releases]
## What's Changed
* Bump squizlabs/php_codesniffer from 3.10.1 to 3.10.2 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4109
* Bump friendsofphp/php-cs-fixer from 3.59.3 to 3.60.0 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4127
* Bump phpstan/phpstan from 1.11.5 to 1.11.7 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4090
* Phpstan fixes by @sreichel in https://github.com/OpenMage/magento-lts/pull/4116
* [DDEV] Adding n98-magerun support for the test environment by @addison74 in https://github.com/OpenMage/magento-lts/pull/4107
* New feature: Added "Newsletter Subscription Form" widget by @empiricompany in https://github.com/OpenMage/magento-lts/pull/4095
* Removed empty `Mage_Poll` config section by @sreichel in https://github.com/OpenMage/magento-lts/pull/4122
* Added support to specify file upload dir in system configuration by @eneiasramos in https://github.com/OpenMage/magento-lts/pull/4079
* Show correct version for main branch by @sreichel in https://github.com/OpenMage/magento-lts/pull/4137
* Allow to use $before parameter in addCss/addJs from XML by @Caprico85 in https://github.com/OpenMage/magento-lts/pull/4151
* Bump phpseclib/phpseclib from 3.0.39 to 3.0.41 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4154
* Bump friendsofphp/php-cs-fixer from 3.60.0 to 3.62.0 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4153
* PHPCS: fixed php compatibility warning by @sreichel in https://github.com/OpenMage/magento-lts/pull/4148
* PHPCS: ignore `mcrypt` related warnings by @sreichel in https://github.com/OpenMage/magento-lts/pull/4147
* Removed MTF by @sreichel in https://github.com/OpenMage/magento-lts/pull/4161
* Updated .gitignore, ref #4161 by @sreichel in https://github.com/OpenMage/magento-lts/pull/4167
* TinyMCE security fix by @sreichel in https://github.com/OpenMage/magento-lts/pull/4157
* Bump friendsofphp/php-cs-fixer from 3.62.0 to 3.64.0 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4178
* PHP-CS-Fixer: added check for double quotes by @sreichel in https://github.com/OpenMage/magento-lts/pull/4126
* Fixed issues #4134 undefined array index label_use_default in Mage_Catalog_Model_Product_Attribute_Backend_Media by @empiricompany in https://github.com/OpenMage/magento-lts/pull/4136
* PhpStan fixes by @sreichel in https://github.com/OpenMage/magento-lts/pull/4146
* PHPCS: fixed last PHPCompatibility check errors by @sreichel in https://github.com/OpenMage/magento-lts/pull/4164
* Small DocBlock update by @sreichel in https://github.com/OpenMage/magento-lts/pull/4160
* Bump phpstan/phpstan from 1.11.7 to 1.12.0 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4177
* PHP-CS-Fixer: added check for short array syntax and logical operators by @sreichel in https://github.com/OpenMage/magento-lts/pull/4181
* Bump tj-actions/changed-files from 44 to 45 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4170
* CodeQL: updated workflow by @sreichel in https://github.com/OpenMage/magento-lts/pull/4172
* PhpStan: bumped to 1.12.1 & renamed config files by @sreichel in https://github.com/OpenMage/magento-lts/pull/4183
* PhpStan: L7 update for `Mage_Sendfriend` by @sreichel in https://github.com/OpenMage/magento-lts/pull/4187
* DDEV: file permission and bugfix by @sreichel in https://github.com/OpenMage/magento-lts/pull/4145
* PHP-CS-Fixer: added check for concat spaces by @sreichel in https://github.com/OpenMage/magento-lts/pull/4185
* Fix strlen() null deprecation in AccountController.php by @kiatng in https://github.com/OpenMage/magento-lts/pull/4155
* Added method to make use of `Mage_Core_Model_Security_HtmlEscapedString` easier by @sreichel in https://github.com/OpenMage/magento-lts/pull/4123
* PHPCS: patched ECG standard by @sreichel in https://github.com/OpenMage/magento-lts/pull/4149
* Added `error` directory to checks + php8.4 fix by @sreichel in https://github.com/OpenMage/magento-lts/pull/4184
* Rector: CQ - UnusedForeachValueToArrayKeysRector by @sreichel in https://github.com/OpenMage/magento-lts/pull/4111
* Bump phpstan/phpstan from 1.12.1 to 1.12.2 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4188
* Worklflow: update copyright monthly by @sreichel in https://github.com/OpenMage/magento-lts/pull/4176
* Add email link to set password. See issue #1205. by @kiatng in https://github.com/OpenMage/magento-lts/pull/3262
* Updated Copyright by @github-actions in https://github.com/OpenMage/magento-lts/pull/4189
* Bump peter-evans/create-pull-request from 6 to 7 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4193
* Fixed undefined array key "values" in Mage_Catalog_Model_Product_Attribute_Backend_Media by @S0FTWEX in https://github.com/OpenMage/magento-lts/pull/4125
* Fix ACL Resource "admin/sales/order/actions/use" not found by @pbaldauf in https://github.com/OpenMage/magento-lts/pull/4199
* Bump symfony/polyfill-php81 from 1.30.0 to 1.31.0 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4195
* Bump symfony/polyfill-php82 from 1.30.0 to 1.31.0 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4194
* Bump symfony/polyfill-php80 from 1.30.0 to 1.31.0 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4197
* PhpUnit: use database for tests, added tests by @sreichel in https://github.com/OpenMage/magento-lts/pull/4138
* Bump symfony/polyfill-php74 from 1.30.0 to 1.31.0 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4198
* Fixed some typos by @sreichel in https://github.com/OpenMage/magento-lts/pull/4200
* Fixed phpstan on Mage_Customer_Model_Customer by @kiatng in https://github.com/OpenMage/magento-lts/pull/4190
* Add new method to get config value directly from DB bypassing cache. by @kiatng in https://github.com/OpenMage/magento-lts/pull/4163
* PHP8: Fixes passing null to `str_contains()` by @sreichel in https://github.com/OpenMage/magento-lts/pull/4203
* Updated `Varien_Object::getData()` and added `getDataByKey()` & `getDataByPath()` by @sreichel in https://github.com/OpenMage/magento-lts/pull/4205
* Rector: DeadCode - removed useless docblocks by @sreichel in https://github.com/OpenMage/magento-lts/pull/4201
* Bump phpstan/phpstan from 1.12.2 to 1.12.3 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4196
* Docs: Added section for PhpStorm (SCSS) file watcher by @sreichel in https://github.com/OpenMage/magento-lts/pull/4208
* PHPUnit: added some tests by @sreichel in https://github.com/OpenMage/magento-lts/pull/4204
* Preview PR in gitpod by @sreichel in https://github.com/OpenMage/magento-lts/pull/4175
* Fixed missing/wrong admin `_setActiveMenu()` by @sreichel in https://github.com/OpenMage/magento-lts/pull/4209
* Fix wrong Preview link on CMS pages grid by @Caprico85 in https://github.com/OpenMage/magento-lts/pull/4214
* Rector: added dependency and basic rules by @sreichel in https://github.com/OpenMage/magento-lts/pull/4213
* In product list all swatches are always shown even if they are not available when "Display Out of Stock Products" is disabled by @empiricompany in https://github.com/OpenMage/magento-lts/pull/4100
* DDEV: command for local development by @sreichel in https://github.com/OpenMage/magento-lts/pull/4133
* Openmage admin theme: Change background-color for active sections by @sreichel in https://github.com/OpenMage/magento-lts/pull/4219
* Backport: Added unittest for date and datetime by @sreichel in https://github.com/OpenMage/magento-lts/pull/4210
* Bump phpseclib/phpseclib from 3.0.41 to 3.0.42 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4222
* Bump squizlabs/php_codesniffer from 3.10.2 to 3.10.3 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4221
* Fixed strcmp(): Passing null to parameter #1 ($string1) of type strin… by @kiatng in https://github.com/OpenMage/magento-lts/pull/4212
* Custom product types - increased column length for widgets by @sreichel in https://github.com/OpenMage/magento-lts/pull/4226
* Skip some rector rules by @sreichel in https://github.com/OpenMage/magento-lts/pull/4227
* Bump composer/composer from 2.7.7 to 2.7.9 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4234
* Updated Copyright by @github-actions in https://github.com/OpenMage/magento-lts/pull/4237
* Admin config: added input validation & hide disabled options by @sreichel in https://github.com/OpenMage/magento-lts/pull/4232
* Some phpStan fixes by @sreichel in https://github.com/OpenMage/magento-lts/pull/4241
* PHP8.1: Added getter method for `Mage_Adminhtml_Block_Widget_Grid_Column::getType()` to return string by @sreichel in https://github.com/OpenMage/magento-lts/pull/4238
* Admin theme: Change background-color for active sections by @sreichel in https://github.com/OpenMage/magento-lts/pull/4246
* Deprecated functionality: preg_match(): Passing null to parameter #4 … by @addison74 in https://github.com/OpenMage/magento-lts/pull/4251
* Bump composer/composer from 2.7.9 to 2.8.1 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4253
* Bugfix: passing invalid ID to `Mage_Catalog_Model_Url::refreshRewrites()` by @sreichel in https://github.com/OpenMage/magento-lts/pull/4259
* Configurable: remove unused double foreach loop by @Hanmac in https://github.com/OpenMage/magento-lts/pull/4008
* Bump phpstan to level5 by @sreichel in https://github.com/OpenMage/magento-lts/pull/4265
* Bump phpstan/phpstan from 1.12.3 to 1.12.6 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4252
* Bump rector/rector from 1.2.5 to 1.2.6 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4254
* Better caching for phpstan workflow by @sreichel in https://github.com/OpenMage/magento-lts/pull/4269
* Added `ext-ftp` to requirements by @sreichel in https://github.com/OpenMage/magento-lts/pull/4270
* Fixes configurable swatch color picker in legacy admin theme by @sreichel in https://github.com/OpenMage/magento-lts/pull/4218
* Better config for admin RSS links by @sreichel in https://github.com/OpenMage/magento-lts/pull/4243
* Bump rector/rector from 1.2.6 to 1.2.7 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4271
* Added polyfill-8.3/8.4 in Composer by @addison74 in https://github.com/OpenMage/magento-lts/pull/4268
* Bump macopedia/phpstan-magento1 from 1.0.7 to 1.0.8 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4272
* Some phpstan fixes by @sreichel in https://github.com/OpenMage/magento-lts/pull/4273
* Fixed incorrect datetime in block, ref #1525 by @sreichel in https://github.com/OpenMage/magento-lts/pull/4242
* Prevent delete/disable CMS pages used in configuration by @sreichel in https://github.com/OpenMage/magento-lts/pull/4239
* Some phpstan fixes by @sreichel in https://github.com/OpenMage/magento-lts/pull/4276
* Updated zf1-future to 1.24.1 by @sreichel in https://github.com/OpenMage/magento-lts/pull/4278
* Phpunit: ignore timezone in date tests by @sreichel in https://github.com/OpenMage/magento-lts/pull/4284
* Some phpstan fixes by @sreichel in https://github.com/OpenMage/magento-lts/pull/4283
* PHP8: base64_decode(): Passing null is deprecated by @sreichel in https://github.com/OpenMage/magento-lts/pull/4281
* Fix placeholder image in WYSIWYG editor when using a different admin theme by @Caprico85 in https://github.com/OpenMage/magento-lts/pull/4240
* Bump phpstan/phpstan from 1.12.6 to 1.12.7 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4300
* Bump rector/rector from 1.2.7 to 1.2.8 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4301
* Unified admin grid columns and added XML configuration by @sreichel in https://github.com/OpenMage/magento-lts/pull/4225
* [PHP 8.4] fgetcsv() - The $escape parameter must be provided as its default value will change by @addison74 in https://github.com/OpenMage/magento-lts/pull/4297
* [PHP 8.4] fputcsv() - The $escape parameter must be provided as its default value will change by @addison74 in https://github.com/OpenMage/magento-lts/pull/4298
* Fixes `Mage_Core_Helper_Url::removeRequestParam()` by @sreichel in https://github.com/OpenMage/magento-lts/pull/4295
* PHP84: ignore deprecated E_STRICT by @sreichel in https://github.com/OpenMage/magento-lts/pull/4289
* Fix deprecated message for strlen first value null by @leissbua in https://github.com/OpenMage/magento-lts/pull/4306
* Added more PhpUnit tests by @sreichel in https://github.com/OpenMage/magento-lts/pull/4236
* Fixed full name observer calls in config.xml by @sreichel in https://github.com/OpenMage/magento-lts/pull/4309
* Bump shardj/zf1-future from 1.24.1 to 1.24.2 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4319
* Add template files to phpstan checks by @sreichel in https://github.com/OpenMage/magento-lts/pull/4245
* Fix `Mage_Adminhtml_Block_Template::isOutputEnabled()` for invalid module by @sreichel in https://github.com/OpenMage/magento-lts/pull/4320
* PHPStan: added some class vars to templates by @sreichel in https://github.com/OpenMage/magento-lts/pull/4322
* Updated Copyright by @github-actions in https://github.com/OpenMage/magento-lts/pull/4327
* Fix: cannot add to wishlist from tag page - Customer account by @sreichel in https://github.com/OpenMage/magento-lts/pull/4329
* PHPStan: some fixes by @sreichel in https://github.com/OpenMage/magento-lts/pull/4328
* Bump pelago/emogrifier from 7.2.0 to 7.3.0 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4332
* Bump composer/composer from 2.8.1 to 2.8.2 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4333
* Bump ezyang/htmlpurifier from 4.17.0 to 4.18.0 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4331
* Hotfix:  `core_store.store_id` can be `null` by @sreichel in https://github.com/OpenMage/magento-lts/pull/4330
* PHPCS-ECG: fail on error by @sreichel in https://github.com/OpenMage/magento-lts/pull/4324
* Bump symfony/process from 5.4.45 to 5.4.46 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4336
* Fix duplicate block rendering with getSortedChildren() by @justinbeaty in https://github.com/OpenMage/magento-lts/pull/4337
* [PHP 8.x] Deprecated functionality: strtotime(): Passing null to para… by @addison74 in https://github.com/OpenMage/magento-lts/pull/4339
* PhpUnit: added tests by @sreichel in https://github.com/OpenMage/magento-lts/pull/4321
* Remove `Mage_Sendfriend` by @sreichel in https://github.com/OpenMage/magento-lts/pull/4274
* Make `Mage_GiftMessage` optional in templates by @Hanmac in https://github.com/OpenMage/magento-lts/pull/4266
* Fixes line-height for some H4 in admin legacy theme by @sreichel in https://github.com/OpenMage/magento-lts/pull/4335
* Code quality: added PHPMD, updated workflow by @sreichel in https://github.com/OpenMage/magento-lts/pull/4303
* Bump rector/rector from 1.2.8 to 1.2.10 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4343
* Bump phpstan/phpstan from 1.12.7 to 1.12.9 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4342
* Bugfix: cant view order in customer account by @sreichel in https://github.com/OpenMage/magento-lts/pull/4345
* More phpstan fixes by @sreichel in https://github.com/OpenMage/magento-lts/pull/4291
* Bugfix: hide exception, ref #4303 by @sreichel in https://github.com/OpenMage/magento-lts/pull/4344
* Documentation v4 📝 - github.io page by @sreichel in https://github.com/OpenMage/magento-lts/pull/4156
* Added `ext-pdo_mysql` to checks by @sreichel in https://github.com/OpenMage/magento-lts/pull/4350
* Fix: create empty SimpleXMLElement by @sreichel in https://github.com/OpenMage/magento-lts/pull/4351
* Bugfix: cant create order from admin backend by @sreichel in https://github.com/OpenMage/magento-lts/pull/4348
* Deprecated functionality: preg_replace() by @addison74 in https://github.com/OpenMage/magento-lts/pull/4311
* Bump composer/composer from 2.8.2 to 2.8.3 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4366
* Bump squizlabs/php_codesniffer from 3.10.3 to 3.11.1 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4367
* [DDEV] Support for `xhprof`/`xhgui` by @sreichel in https://github.com/OpenMage/magento-lts/pull/4325
* Fixed null deprecation in Varien/Convert/Parser/Xml/Excel by @kiatng in https://github.com/OpenMage/magento-lts/pull/4375
* Fixed null deprecation in `Mage_Catalog_Model_Product_Option_Type_Text::validateUserValue()` by @kiatng in https://github.com/OpenMage/magento-lts/pull/4357
* Bugfix: `Mage_Adminhtml_Helper_Dashboard_Data::countStores()` on null by @sreichel in https://github.com/OpenMage/magento-lts/pull/4370
* Fix validateHttpVersion to be RFC 1945 and RFC 7230 compliant by @leissbua in https://github.com/OpenMage/magento-lts/pull/4368
* PHPUnit: added tests by @sreichel in https://github.com/OpenMage/magento-lts/pull/4340
* Bump macopedia/phpstan-magento1 from 1.0.8 to 1.1.0 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4379
* Bump friendsofphp/php-cs-fixer from 3.64.0 to 3.65.0 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4380
* PHPStan: some fixes by @sreichel in https://github.com/OpenMage/magento-lts/pull/4346
* Bugfix: fixed sitemap config depend nodes by @sreichel in https://github.com/OpenMage/magento-lts/pull/4369
* Fixed issue #4384 bug on not able to view order details in frontend. by @kiatng in https://github.com/OpenMage/magento-lts/pull/4385
* Cache `isModuleOutputEnabled` or `isModuleEnabled` status by @sreichel in https://github.com/OpenMage/magento-lts/pull/4323
* Obfuscated login credentials in stacktraces by @sreichel in https://github.com/OpenMage/magento-lts/pull/4388
* PhpStan: some fixes by @sreichel in https://github.com/OpenMage/magento-lts/pull/4363
* Rector: updated UnitTests by @sreichel in https://github.com/OpenMage/magento-lts/pull/4391
* Fix for catalog_product_relation not being deleted when bundle option deleted. by @dannydnz in https://github.com/OpenMage/magento-lts/pull/4395
* Rector: set methods to private in `Mage.php` by @sreichel in https://github.com/OpenMage/magento-lts/pull/4392
* Added test, ref #4352 by @sreichel in https://github.com/OpenMage/magento-lts/pull/4389
* Use `transliterator_transliterate` to generate "url_key" by @sreichel in https://github.com/OpenMage/magento-lts/pull/4315
* Fix unserializeArray on nonserialized string by @Hanmac in https://github.com/OpenMage/magento-lts/pull/4387
* PHPMD: remove unused private methods/fields by @sreichel in https://github.com/OpenMage/magento-lts/pull/4397
* DOCs: update by @sreichel in https://github.com/OpenMage/magento-lts/pull/4349
* Updated Copyright by @github-actions in https://github.com/OpenMage/magento-lts/pull/4400
* Bump actions/setup-python from 4 to 5 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4365
* Cleanup:  removed broken `Mage_Core_Controller_Front_Router` by @sreichel in https://github.com/OpenMage/magento-lts/pull/4356
* Remove gitpod files by @sreichel in https://github.com/OpenMage/magento-lts/pull/4404
* Rector: removed useless variables by @sreichel in https://github.com/OpenMage/magento-lts/pull/4393
* New feature: Highlight popular countries in country select by @ma4nn in https://github.com/OpenMage/magento-lts/pull/4089
* Install TinyMCE 7.5 via composer by @sreichel in https://github.com/OpenMage/magento-lts/pull/4113
* DDEV: updated install script by @sreichel in https://github.com/OpenMage/magento-lts/pull/4407
* Follow coding-standard PER-CS2.0 by @sreichel in https://github.com/OpenMage/magento-lts/pull/4372
* Bump tinymce/tinymce from 7.5.1 to 7.6.0 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4422
* Bump mklkj/tinymce-i18n from 24.11.25 to 24.12.9 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4423
* Bump squizlabs/php_codesniffer from 3.11.1 to 3.11.2 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4420
* Bump phpseclib/phpseclib from 3.0.42 to 3.0.43 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4421
* Bump composer/composer from 2.8.3 to 2.8.4 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4419
* PHPCS: merged workflows by @sreichel in https://github.com/OpenMage/magento-lts/pull/4416
* Readme: fixed badges by @sreichel in https://github.com/OpenMage/magento-lts/pull/4413
* [Tax] add FK to tax/sales_order_tax again by @Hanmac in https://github.com/OpenMage/magento-lts/pull/4334
* Fix Possible Array Access warnings in Captcha Observer by @Hanmac in https://github.com/OpenMage/magento-lts/pull/4424
* Fixed null deprecation array_keys in Multishipping. by @kiatng in https://github.com/OpenMage/magento-lts/pull/4418

## New Contributors
* @github-actions made their first contribution in https://github.com/OpenMage/magento-lts/pull/4189
* @S0FTWEX made their first contribution in https://github.com/OpenMage/magento-lts/pull/4125
* @pbaldauf made their first contribution in https://github.com/OpenMage/magento-lts/pull/4199
* @dannydnz made their first contribution in https://github.com/OpenMage/magento-lts/pull/4395

**Full Changelog**: https://github.com/OpenMage/magento-lts/compare/v20.10.1...v20.11.0

<hr /><em>This discussion was created from the release <a href='https://github.com/OpenMage/magento-lts/releases/tag/v20.11.0'>v20.11.0</a>.</em>

**Comment by sreichel** [comment]
Thanks to all contributors and reviewers!

Happy X-mas :christmas_tree:

**Reply by eneiasramos** [reply]
Hohoho Merry Christmas! 🎄🎅

**Comment by ajbonner** [comment]
Amazing work - thank you so much to all involved!

**Comment by S0FTWEX** [comment]
Well done! Keep it up!  And don't forget to release also version 21.0.0-beta3 or directly 21.0.0. ;-)
I'm using 21.0.0-beta2 in my production and I want to have this release but don't want to downgrade :-(

**Reply by sreichel** [reply]
Sorry for sayin that, but i doubt there will be an update for v21 soon.

I dont want to work on it b/c the time to spent on it outweight the benefits.

If you really want it, please consider sponsorhip via opencollective.org or paypal.

**Comment by alexh-swdev** [comment]
Thanks. May keep me busy "between the years" ;)

**Comment by toejam34** [comment]
Random observation - you used to name your zips openmage-version.zip example : openmage-v20.10.2.zip

This one is named magento-lts-20.11.0.zip

Is there any reason for this?

One other thing I have spotted - I have unzipped the file to my testing server but the version at the bottom of the CP does not appear to have bumped : 

![image](https://github.com/user-attachments/assets/bc2c7795-cad6-4a13-8fc8-f91cd1041d8b)

Edit - spotted why - unzipped to a subfolder and not into the main folder

**Comment by imavisionro** [comment]
Hi there,
I can`t find vendor folder in the new version and i receive an error after upgrade

**Reply by sreichel** [reply]
Looks like you have downloaded the repo zip-archive?

**Reply by imavisionro** [reply]
I download the latest version, https://github.com/OpenMage/magento-lts/archive/refs/tags/v20.11.0.tar.gz
